airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -508,17 +522,21 @@ def await_container_completion(self, pod: V1Pod, container_name: str) -> None:
             self.log.info("Waiting for container '%s' state to be completed", container_name)
             time.sleep(1)
 
-    def await_pod_completion(self, pod: V1Pod) -> V1Pod:
+    def await_pod_completion(self, istio_enabled: bool, pod: V1Pod, container_name: str) -> V1Pod:
```

Comment:
NIT. Can we leave the `pod` as first value, make the other two optional ? That sounds much more logical and will still allow to use the util without sprinkling istio everywhere?

Reply:
Thanks. Sure I've make `pod` arg as first value and make the other two optional with default value if not specified